sklearn/tests/test_common.py

```diff
@@ -644,3 +644,24 @@ def test_global_output_transform_pandas(estimator):
     _set_checking_parameters(estimator)
     with ignore_warnings(category=(FutureWarning)):
         check_global_ouptut_transform_pandas(estimator.__class__.__name__, estimator)
+
+
+STATELESS_ESTIMATORS = [
+    est for est in _tested_estimators() if _safe_tags(est, key="stateless")
+]
+
+
+@pytest.mark.parametrize(
+    "estimator", STATELESS_ESTIMATORS, ids=_get_check_estimator_ids
+)
+def test_no_fitted_error_stateless_estimator(estimator):
```

Comment:
I would write this test differently. It should be a `check` in the `estimator` check and should be added in `_yield_all` with a `if "stateless" is tags` (or something equivalent).